src/Dispatch/__init__.py

```diff
@@ -1,2 +1,13 @@
 # Copyright 2020, Battelle Energy Alliance, LLC
 # ALL RIGHTS RESERVED
+
+import importlib
+import os
+import sys
+
+sys.path.append(os.path.join(os.path.dirname(__file__), '..', '..', '..'))
```

Comment:
One suggestion for these multiple '..' inputs could be to replace with `os.pardir`. Unsure if it's cleaner but this line, where we add the parent directory that is 4 directories up, could look something like: `sys.path.append(os.path.join(__file__, *[os.pardir]*4))`